ivadomed_singularity_master.def

```diff
@@ -0,0 +1,71 @@
+BootStrap: docker
+From: python:3.8-slim-bullseye
```

Comment:
We'd need some testing to finalize the base image taking into account the usage of GPU... So, it primarily boils down to choosing one out of the following: 
a. `python:3.8-slim-bullseye`
b. `pytorch/pytorch:1.11.0-cuda11.3-cudnn8-devel` or
c. `nvidia/cuda:11.7.1-cudnn8-runtime-ubuntu20.04`